### DISCUSSION TITLE
How can we create custom header with our own requirements?

### DISCUSSION THREAD

**Discussion by ParulVermaVT** [kvyatkovskys/KVKCalendar#348] [Q&A]
I am trying to create custom headers.
I've used this method from the datasource but it does not seems affecting the UI.

`func dequeueHeader<T>(date: Date?, type: CalendarType, view: T, indexPath: IndexPath) -> KVKCalendarHeaderProtocol? where T : UIScrollView`

![Screenshot 2024-04-25 at 11 21 06 AM](https://github.com/kvyatkovskys/KVKCalendar/assets/148210649/fb7773ed-a8fe-4a8c-a780-c5b8f88b819a)

**Comment by ParulVermaVT** [comment]
__

**Reply by kvyatkovskys** [reply]
Hi @ParulVermaVT, I’ll check this flow on this week, thx

**Reply by ParulVermaVT** [reply]
Hi @kvyatkovskys 
Thanks for the response.
1. I've managed it using custom xib under this delegate 
`func dequeueCell<T>(parameter: CellParameter, type: CalendarType, view: T, indexPath: IndexPath) -> KVKCalendarCellProtocol? where T: UIScrollView`.

2. I've another requirement of scroll to an event vertically.
I managed to scroll to a particular date but did not find any method to scroll to a particular event under a specific date.
Please let me know how can i achieve this feature
